### DISCUSSION TITLE
part-ii/2014-61/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#289] [Question comments]
Part II 2014 Galois Theory: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2014-61/

**Comment by DannyDuberstein** [comment]
(i) f min poly of x / K, f insep so f(T) = g(T^p) = h(T)^p for some h in F_p(t^1/p), then tower law on K(x,t^1/p).
(ii) Take n maximal s.t. t^(1/(p^n))=y is in L. Then follows from first part L/K(y) is separable as otherwise get a p^th root of y in L.

Usual suspects Gauss+Eisenstein for irreducibility of both the quadratic and quartic. Can get roots of quartic by two quadratic extensions, so degree of sf is exactly 4.
